api/src/opentrons/protocol_engine/commands/dispense_in_place.py

```diff
@@ -23,7 +23,9 @@
 class DispenseInPlaceParams(PipetteIdMixin, VolumeMixin, FlowRateMixin):
     """Payload required to dispense in place."""
 
-    pass
+    pushOut: Optional[float] = Field(
+        None, description="perform a small blow out for accurate dispensing"
```

Comment:
```suggestion
        None, description="push the plunger a small amount farther than necessary for accurate low-volume dispensing"
```
i guess? @andySigler what do you think